src/main/java/com/cocodan/triplan/post/schedule/dto/request/SchedulePostLikeRequest.java

```diff
@@ -0,0 +1,13 @@
+package com.cocodan.triplan.post.schedule.dto.request;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class SchedulePostLikeRequest {
+
+    private Long schedulePostId;
+
+    private Boolean flag;
```

Comment:
```suggestion
    private long schedulePostId;

    private boolean flag;
```
primitive로 바꾸는거 어떠신가요??

Reply:
id와 관련된 경우에는 primitive type을 쓰면 값이 안 들어왔을 때에도 id 0 을 가지는 엔티티를 찾게되어서 Wrapper class가 더 적절할거같다고 생각했습니다.

Boolean 같은 경우에는 lombok에서 Boolean은 getXXX()을, boolean은 isXXX() 을 지원해주기 때문에 getFlag() 메서드를 이용하기 위해 Boolean으로 설정했습니다.

Reply:
오홍 그렇군요. getX, isX 로 만들어 주는건 처음 알았네요.. 굿굿입니다~~~ 👍